### PR TITLE
Updating to WZDx v4.1

### PR DESCRIPTION
**Pull request by jacob6838**
Main updates:
- ids to UUIDs
- name property for human-friendly names
- Direction supports undefined, unknown
- no-passing restriction type
- road_event_feed_info -> feed_info
- verification/accuracy is reworked to booleans
- remove event_status
- road_names -> optional
- relationship -> related_road_events property

Other changes:
Adding experimental feed (wanted to update that to 4.1 as well)

### CODE REVIEW

File: process_logs.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import csv
```
Comment (payneBrandon) [inline process_logs.py]: What is this process_logs and the associated csv for?

Comment (payneBrandon) [reply process_logs.py]: And the no_start_dates.json? Are these just bunk test files at this point?

File: tests/data/experimental_combination/icone/wzdx_1.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
```
Comment (payneBrandon) [inline tests/data/experimental_combination/icone/wzdx_1.json]: Are these "wzdx_xxx" named files the planned events we want to combine with? Should we name them something other than wzdx_1/2/3 to be more specific about what data we're testing with?

File: wzdx/sample_files/README.md [hunk 3]
```diff
@@ -10,6 +10,6 @@ Output Message : WZDx message Generation Output
 
 validation_schema : wzdx schema(from specification)
 
--wzdx_v3.0_feed.json
--wzdx_v3.1_feed.json
--wzdx_v4.0_feed.json
+- wzdx_v3.1_feed.json
```
Comment (payneBrandon) [inline wzdx/sample_files/README.md]: Looks like this v3.1 has been removed, should we drop it from this README as well?

File: wzdx/sample_files/enhanced/icone/icone.json [hunk 4]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (payneBrandon) [inline wzdx/sample_files/enhanced/icone/icone.json]: Is this file being used? It looks to be version 4.0 so maybe being deprecated and can be removed?

File: wzdx/sample_files/enhanced/icone/incidents_extended.json [hunk 5]
```diff
@@ -0,0 +1,476 @@
+{
```
Comment (payneBrandon) [inline wzdx/sample_files/enhanced/icone/incidents_extended.json]: Same here, is this one being used? I don't see a reference anywhere to it.

File: wzdx/sample_files/enhanced/planned_events/prod_snapshot_2022_09_23.json [hunk 6]
```diff
@@ -0,0 +1,6407 @@
+{
```
Comment (payneBrandon) [inline wzdx/sample_files/enhanced/planned_events/prod_snapshot_2022_09_23.json:1]: Do we need this snapshot for some test?

File: wzdx/sample_files/enhanced/version_4_1.json [hunk 7]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (payneBrandon) [inline wzdx/sample_files/enhanced/version_4_1.json]: File doesn't seem to be referenced anywhere.

File: wzdx/sample_files/raw/icone/incidents_extended.json [hunk 8]
```diff
@@ -0,0 +1,476 @@
+{
```
Comment (payneBrandon) [inline wzdx/sample_files/raw/icone/incidents_extended.json]: Not referenced anywhere, can we drop?

File: wzdx/standard_to_enhanced/Untitled-1.json [hunk 9]
```diff
@@ -0,0 +1 @@
...ero/1.0/", "data_sources": [{"data_source_id": "w", "organization_name": "CDOT", "update_date": "2021-04-13T00:00:00Z", "update_frequency": 300, "contact_name": "Ashley Nylen", "contact_email": "ashley.nylen@state.co.us"}], "update_date": "2021-04-13T00:00:00Z", "update_frequency": 300, "contact_name": "Ashley Nylen", "contact_email": "ashley.nylen@state.co.us"}, "type": "FeatureCollection", "features": [{"id": "1245", "type": "Feature", "properties": {"core_details": {"event_type": "work-zone", "data_source_id": "w", "road_names": ["I-75 N"], "direction": "northbound", "description": "19-1245: Roadwork between MP 40 and MP 48", "creation_date": "2019-11-05T01:22:20Z", "update_date": "2021-11-05T19:56:03Z"}, "start_date": "2021-06-08T20:15:01Z", "is_start_date_verified": false, "is_end_date_verified": False, "is_start_position_verified": False, "is_end_position_verified": False, "location_method": "channel-device-method", "vehicle_impact": "all-lanes-open"}, "geometry": {"type": "LineString", "coordinates": [[-84.1238971, 37.1686478], [-84.1238971, 37.1686478], [-84.145861, 37.1913], [-84.145861, 37.1913], [-84.157105, 37.201197], [-84.167033, 37.206079], [-84.204074, 37.21931]]}}]
```
Comment (payneBrandon) [inline wzdx/standard_to_enhanced/Untitled-1.json]: I assume we can drop?

File: wzdx/sample_files/enhanced/icone/icone.json (hunk 4 above)
Comment (jacob6838) [reply wzdx/sample_files/enhanced/icone/icone.json]: removed

File: wzdx/sample_files/README.md (hunk 3 above)
Comment (jacob6838) [reply wzdx/sample_files/README.md]: removed

File: wzdx/sample_files/enhanced/icone/incidents_extended.json (hunk 5 above)
Comment (jacob6838) [reply wzdx/sample_files/enhanced/icone/incidents_extended.json]: removed

File: wzdx/sample_files/enhanced/planned_events/prod_snapshot_2022_09_23.json (hunk 6 above)
Comment (jacob6838) [reply wzdx/sample_files/enhanced/planned_events/prod_snapshot_2022_09_23.json:1]: Nope, I removed it, but I left the most recent one (always useful to have an up to date sample)

File: wzdx/sample_files/enhanced/version_4_1.json (hunk 7 above)
Comment (jacob6838) [reply wzdx/sample_files/enhanced/version_4_1.json]: removed

File: wzdx/sample_files/raw/icone/incidents_extended.json (hunk 8 above)
Comment (jacob6838) [reply wzdx/sample_files/raw/icone/incidents_extended.json]: Removed

File: wzdx/standard_to_enhanced/Untitled-1.json (hunk 9 above)
Comment (jacob6838) [reply wzdx/standard_to_enhanced/Untitled-1.json]: Removed

File: tests/data/experimental_combination/icone/wzdx_1.json (hunk 2 above)
Comment (jacob6838) [reply tests/data/experimental_combination/icone/wzdx_1.json]: Renamed and updated tests

**Pull request merged**